qiime2/citations.bib

```diff
@@ -1,11 +1,12 @@
-@article{caporaso2010qiime,
-  title={QIIME allows analysis of high-throughput community sequencing data},
-  author={Caporaso, J Gregory and Kuczynski, Justin and Stombaugh, Jesse and Bittinger, Kyle and Bushman, Frederic D and Costello, Elizabeth K and Fierer, Noah and Peña, Antonio Gonzalez and Goodrich, Julia K and Gordon, Jeffrey I and Huttley, Gavin A and Kelley, Scott T and Knights, Dan and Koenig, Jeremy E and Ley, Ruth E and Lozupone, Catherine A and McDonald, Daniel and Muegge, Brian D and Pirrung, Meg and Reeder, Jens and Sevinsky, Joel R and Turnbaugh, Peter J and Walters, William A and Widmann, Jeremy and Yatsunenko, Tanya and Zaneveld, Jesse and Knight, Rob},
-  journal={Nature methods},
-  volume={7},
-  number={5},
-  pages={335},
-  year={2010},
-  publisher={Nature Publishing Group},
-  doi={10.1038/nmeth.f.303}
+@article{10.7287/peerj.preprints.27295v1,
```

Comment:
Missing test updates in `qiime2.core.archive.tests.test_citations.TestCitationsTracked` (`test_action` and `test_import`).

Reply:
>_< thanks!